keps/sig-node/2371-cri-pod-container-stats/README.md

```diff
@@ -669,6 +691,9 @@ the health of the service?**
     job creation time) for cron job <= 10%
   - 99,9% of /health requests per day finish with 200 code
 
+- 100% coverage of metrics reported by cAdvisor reported by CRI
```

Comment:
I think some of the metrics we said may not be needed to promising 100% coverage may be a bit too optimistic :)

Reply:
alright I dropped it, though I think we may need to provide close to it..